tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,334 @@
+        mt = mt.annotate_cols(
+            normalized_y_coverage=hl.or_missing(
+                mt[f"chr{normalization_contig}_mean_dp"] > 0,
+                mt.chrY_mean_dp / mt[f"chr{normalization_contig}_mean_dp"],
+            )
+        )
+        if add_x_cov:
+            chrx_ht = get_chr_cov(mt, "GRCh38", "X", call_rate_threshold)
+            mt = mt.annotate_cols(**chrx_ht[mt.col_key])
+            mt = mt.annotate_cols(
+                normalized_x_coverage=hl.or_missing(
+                    mt[f"chr{normalization_contig}_mean_dp"] > 0,
+                    mt.chrX_mean_dp / mt[f"chr{normalization_contig}_mean_dp"],
+                )
+            )
+        sex_ht = run_hails_impute_sex(
+            mt,
+            build,
+            outdir,
+            mt_name,
+            xy_fstat_threshold,
+            xx_fstat_threshold,
+            aaf_threshold,
+        )
+        sex_ht = sex_ht.annotate(
+            ambiguous_sex=hl.is_missing(sex_ht.is_female),
+            sex_aneuploidy=(sex_ht.is_female)
+            & hl.is_defined(sex_ht.normalized_y_coverage)
+            & (sex_ht.normalized_y_coverage > y_cov_threshold)
+            | (~sex_ht.is_female)
+            & hl.is_defined(sex_ht.normalized_y_coverage)
+            & (sex_ht.normalized_y_coverage < y_cov_threshold),
+        )
+
+        sex_expr = (
+            hl.case()
+            .when(sex_ht.ambiguous_sex, "ambiguous_sex")
+            .when(sex_ht.sex_aneuploidy, "sex_aneuploidy")
+            .when(sex_ht.is_female, "XX")
+            .default("XY")
+        )
+        sex_ht = sex_ht.annotate(sex=sex_expr)
+
+    else:
+        sex_ht = run_hails_impute_sex(
+            mt,
+            build,
+            outdir,
+            mt_name,
+            xy_fstat_threshold,
+            xx_fstat_threshold,
+            aaf_threshold,
+        )
+        sex_ht = sex_ht.annotate(ambiguous_sex=hl.is_missing(sex_ht.is_female))
+        sex_expr = hl.if_else(
+            sex_ht.ambiguous_sex,
+            "ambiguous_sex",
+            hl.if_else(sex_ht.is_female, "XX", "XY"),
+        )
+        sex_ht = sex_ht.annotate(sex=sex_expr)
+
+    outfile = f"{outdir}/sex_{mt_name}.txt"
+    sex_ht.export(outfile)
+    return sex_ht
+
+
+def main(args):
+    """
+    Call `call_sex` to infer sex of samples in input MatrixTable.
+
+    :param args: User's command line inputs
+    """
+
+    call_sex(**vars(args))
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="This script infers the sex of samples"
+    )
+    parser.add_argument(
+        "-i", "--callset", required=True, help="Path to Callset MatrixTable"
+    )
+    parser.add_argument(
+        "-u",
+        "--use-y-cov",
+        help="Whether to use chromosome Y coverage when inferring sex. Note that Y coverage is required to infer sex aneuploidies",
+        action="store_true",
+    )
+    parser.add_argument(
+        "-x",
+        "--add_x_cov",
```

Comment:
```suggestion
        "--add-x-cov",
```
we try to use hyphens when specifying arguments